src/SWP/Bundle/ContentBundle/Loader/ArticleLoader.php

```diff
@@ -79,8 +79,32 @@ public function load($type, $parameters = [], $withoutParameters = [], $response
 
             try {
                 $article = $this->articleProvider->getOneByCriteria($criteria);
-
-                return $this->getArticleMeta($article);
+                $articleBody = $article->getBody();
+                $metaArticle = $this->getArticleMeta($article);
```

Comment:
Don't generate meta and then update article - update article and and when all is done - get meta from it.